doc/src/records/accumulator.md

```diff
@@ -6,12 +6,26 @@ TODO: expand description and example
 
 ACCUMULATOR results must be reported using user-defined REPORTs or EXPORTs.  For example --
 
-    REPORT rpType=UDT rpFreq=Month rpDayBeg=Jan 1 rpDayEnd=Dec 31
-        REPORTCOL colHead="mon" colVal=$Month colWid=3
-        REPORTCOL colHead="Total" colVal=@Accumulator[ 1].M.acmSum colDec=0 colWid=10
-        REPORTCOL colHead="Average" colVal=@Accumulator[ 1].M.acmMean colDec=0 colWid=10
+    ACCUMULATOR "Zone Sensible Cooling Rate [Btu/h]" 
+      acmValue = @znRes[ 1].H.qscHvac // Hourly Output
 
 
+    REPORT rpType=UDT rpFreq=Month rpDayBeg=Jan 1 rpDayEnd=Dec 31 // Monthly Report
+        REPORTCOL colHead="Month"                                          colVal=$Month                                                                   colWid= 3
+        REPORTCOL colHead="Monthly Total Cooling [Btu]"                    colVal=@Accumulator["Zone Sensible Cooling Rate [Btu/h]"].M.acmSum    colDec=0  colWid=10
+        REPORTCOL colHead="Monthly Average Hourly Cooling Rate [Btu/h]"    colVal=@Accumulator["Zone Sensible Cooling Rate [Btu/h]"].M.acmMean   colDec=0  colWid=10
+        REPORTCOL colHead="Monthly Max Hourly Cooling Rate [Btu/h]"        colVal=@Accumulator["Zone Sensible Cooling Rate [Btu/h]"].M.acmMax    colDec=0  colWid=10
+        REPORTCOL colHead="Monthly Min Hourly Cooling Rate [Btu/h]"        colVal=@Accumulator["Zone Sensible Cooling Rate [Btu/h]"].M.acmMin    colDec=0  colWid=10
+
+    ACCUMULATOR "Zone Solar Gain [Btu]" 
+      acmValue = @znRes[ 1].S.qSlr // Subhourly Output
```

Comment:
Similar question here. I'm not sure you are using the correct units / interpretation here.